myhpi/search/templates/search/search_result.html

```diff
@@ -3,7 +3,12 @@
 
 <a href="{% pageurl result %}">
     <div class="search-result">
-        <h5>{{ result.title }}</h5>
+        <hgroup class="d-flex justify-content-between">
+            <h5 class="text-break">{{ result.title }}</h5>
+            <time datetime="{{ result.specific.date|date:"c" }}">
+                {{ result.specific.date|date:"SHORT_DATE_FORMAT" }}
```

Comment:
Since other pages don't have date this is now only for minutes? Intentional?

Reply:
Yes, see <https://github.com/fsr-de/myHPI/pull/492#discussion_r1490861347>